python/lsst/ts/observing/block.py

```diff
@@ -198,7 +198,18 @@ def get_script_configuration(self) -> str:
         config : `str`
             Script configuration.
         """
-        return yaml.safe_dump(self.parameters) if self.parameters else ""
```

Comment:
Is it possible that the problem is that parameters is defined as `dict[str, Any]` and that if it was restricted to the plain types we expect to be used that things would work without this change?

Reply:
I was able to reproduce this issue standalone, just by getting a dictionary and doing the round trip, so I don't think so. 

Also I think the type annotation is ignored altogether by the runtime, no?

Reply:
The type annotation is critical for Pydantic. It coerces types and validates that they match. It's an important part of how pydantic works. If you know that the parameters are string or numbers and not arbitrary python types then we should probably say that regardless of this YAML bug.

I think you are saying that YAML writes out a sexagesimal string incorrectly? Can you put the actual error on the Jira ticket?

Reply:
It is pretty simple to reproduce it. I will add it to the ticket.

```
In [1]: import sys

In [2]: import yaml

In [3]: a = dict(object_ra="+0:31:18")

In [4]: b = dict(object_ra="+1:31:18")

In [5]: yaml.safe_dump(a, sys.stdout)
object_ra: +0:31:18

In [6]: yaml.safe_dump(b, sys.stdout)
object_ra: '+1:31:18'
```